drivers/adc/adc_stm32.c

```diff
@@ -650,12 +762,13 @@ static void adc_stm32_isr(const struct device *dev)
 	const struct adc_stm32_cfg *config =
 		(const struct adc_stm32_cfg *)dev->config;
 	ADC_TypeDef *adc = config->base;
+	int adcData = LL_ADC_REG_ReadConversionData32(adc);
```

Comment:
We need to follow Zephyr convention here
```suggestion
	int adc_data = LL_ADC_REG_ReadConversionData32(adc);
```

Is there a reason why this is always read during an ISR? And not only when `data->buffer[data->samples_count++] =`? If so, then add a comment above the line saying "Read to clear the read flag" or something.

Reply:
Just a leftover, as this was used just to log the value
fixed